tests/common/plugins/conditional_mark/issue.py

```diff
@@ -69,8 +71,29 @@ def is_active(self):
         Returns:
             bool: False if the issue is closed else True.
         """
+        INV_ENV_FILE = '../../../../ansible/group_vars/{}/env.yml'.format(self.inv_name)
+        PUBLIC_ENV_FILE = '../../../../ansible/group_vars/all/env.yml'
+        base_path = os.path.dirname(__file__)
+        if os.path.exists(INV_ENV_FILE):
+            env_file_path = os.path.join(base_path, INV_ENV_FILE)
+        else:
+            env_file_path = os.path.join(base_path, PUBLIC_ENV_FILE)
```

Comment:
The env file could exists but have empty content (probably comment only). This logic here is not robust enough.

Reply:
fixed